dir.props

```diff
@@ -131,6 +131,7 @@
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)</IntermediateOutputPath>
 
     <RuntimePath Condition="'$(RuntimePath)' == ''">$(IntermediateOutputRootPath)test-runtime/</RuntimePath>
+    <CoreClrRuntimePath Condition="'$(CoreClrRuntimePath)' == ''">$(IntermediateOutputRootPath)CoreClrRuntime/</CoreClrRuntimePath>
```

Comment:
Is this used by the product build? If not, maybe move to somewhere under `/tests/` so that it doesn't complicate the product build?

Reply:
Removed this since it's not used any anything other than the test build.